index.html

```diff
@@ -5,175 +5,167 @@
+                        <p class="card-paragraph"> In relation to children, the creative arts are activities that engage
+                            a child's imagination and
+                            can include activit.</p>
+
                     </div>
-                <div class="card-item-child-featured">
-                    <div class="portfolio"><img src="images/portfolio.png" alt="" class="port"></div>
-                    <div class="card-one">
-                    <h3 class="lecture">Alfred KInyua</h3>
-                    <!-- <div class="paragraphs"> -->
-                    <p  class="red-paragraph">Lorem ostrum, culpa facere nisi sapiente perspiciatis, accusantium do
-                         voluptatum dolore.</p>
-                    <p class="car-paragraph">Lorem ostrum, culpa facere nisi sapiente perspiciatis, accusantium do
+                    <div class="card-item-child">
+                        <div><span><i class="fa-solid fa-images pic"></i></span></div>
+                        <h3 class="child-art">Graphics</h3>
+                        <p class="card-paragraph">Lorem ostrum, culpa facere nisi sapiente perspiciatis, accusantium do
                             voluptatum dolore.</p>
+
                     </div>
                 </div>
+                <div class="buttons">
+                    <button class="btn mobile"> Join the Art club today</button>
                 </div>
+                <div class="whole-program desktop">
+                    <a href="#">SEE THE WHOLE PROGRAM</a>
                 </div>
-                <div class="more mobile">
-                    <h4>MORE</h4>
-                    <div><span><i class="fa-solid fa-chevron-down"></i></span></div>
-                </div>
-            </section> 
-            <footer>
-                <img src="images/kenya.png" alt="">
-                <div class="footer-content">
-                <h4>@ Lorem ipsum dolor sit amet consectetur adipisicing elit.
-                     Ute minus illo animi iusto ipsam fugiat.
-                </h4>
-                <h4>Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit, vero.</h4>
-                </div>
-            </footer>
+            </div>
+
+        </section>
+        <section class="featured">
+            <h3 class="heading-three-cards featured-h3">Featured Speakers</h3>
+            <hr id="center">
+           
+            <div class="grid-card">
+                <section class="grid-card featured-section"></div>
+            </div> 
+
+            <div class="more mobile">
+                <h4>MORE</h4>
+                <div><span><i class="fa-solid fa-chevron-down"></i></span></div>
+               
+        
+                
+            </div>
+        </section>
+        <div class="top-footer">            
+            <div class="footer-content desktop">
+            <h3>Partnership</h3>
+            <hr id="Partnership">
```

Comment:
- [ ] [OPTIONAL] The color for the Partnership heading according to the design specs should be `#d3d3d3` not white. This is optional because the previous reviewers missed it 👍

Reply:
Thanks, ill fix it immediately.